analysers/commons/commons.py

```diff
@@ -159,7 +159,7 @@ def computeLevene(sc, cassandraKeyspace, expTable, dataTable, experimentID, cont
             .map(lambda a: a["trial_id"]) \
             .distinct() \
             .collect()
-    if len(trials) < 2:
+    if len(trials) < 3:
```

Comment:
@VincenzoFerme Can give a NaN when lower than 3, so will need at least 3 to have something meaningful here.